.editorconfig

```diff
@@ -17,3 +17,11 @@ trim_trailing_whitespace = false
 
 [*.yml]
 indent_size = 2
+
+[*.php]
+ij_php_align_phpdoc_comments = false
```

Comment:
Didn't know it was possible to specify that here for intellij/phpstorm ! :+1:

Reply:
Not sure if it is supported before phpstorm 2019.2, but in that version it is at least able to add that in the editorconfig file rather easily. :)